src/utils/cache.rs

```diff
@@ -38,6 +38,11 @@ impl Cache for ServiceCache {
 }
 
 pub trait CacheExt: Cache {
+    fn invalidate_caches(&self, key: &str) {
+        self.invalidate_pattern(&format!("c_resp_*{}*", &key));
+        self.invalidate_pattern(&format!("c_reqs_*{}*", &key));
```

Comment:
Just as an idea, maybe we could have a bit of a bigger cache manager struct, that keeps track of all the constants the cache handles, then we could have more specific methods for invalidating specific parts of the cache.

Reply:
I like this direction though, having a prefix that would allow distinguishing between cached request and responses.

Reply:
I tried to keep these keys already in the cache module. Should we extract them to constants?